packages/terra-form-select/tests/jest/SelectOption.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Option from '../../src/SelectOption';
+
+it('should render a default Option component', () => {
+  const option = <Option value="a" />;
+  const wrapper = shallow(option);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render an Option component with display', () => {
+  const option = <Option value="a" display="Apple" />;
+  const wrapper = shallow(option);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a selected Option', () => {
+  const option = <Option value="a" display="Apple" isSelected />;
+  const wrapper = shallow(option);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a disabled Option', () => {
+  const option = <Option value="a" display="Apple" disabled />;
+  const wrapper = shallow(option);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render an Option with children', () => {
```

Comment:
Maybe add a test that has children & a display to confirm only display is rendered as described in the children description?

Reply:
Added test: https://github.com/cerner/terra-core/pull/978/commits/70812139c85b71e4035c0388406c73685de32e1f